daiquiri/__init__.py

```diff
@@ -105,6 +106,21 @@ def logging_excepthook(exc_type, value, tb):
         logging.captureWarnings(True)
 
 
+def setup_colors(palette=formatter.Swatches.Default):
```

Comment:
I am not sure how this is useful. Maybe its usecase must be documented somewhere?